apis/apps/v1beta3/emqxplugin_webhook.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta3
+
+import (
+	"errors"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// log is for logging in this package.
+var emqxpluginlog = logf.Log.WithName("emqxplugin-resource")
+
+var notAllowedPlugin = []string{"emqx_management", "emqx_dashboard"}
```

Comment:
```suggestion
const notAllowedPlugin = []string{"emqx_management", "emqx_dashboard"}
```

Reply:
the data types in constants can only be boolean, numeric (integer, float, and complex) and string.